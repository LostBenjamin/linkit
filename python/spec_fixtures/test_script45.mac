
$a 1
$b `3 if <a> == 1 else 7`

[c]
  <b>

