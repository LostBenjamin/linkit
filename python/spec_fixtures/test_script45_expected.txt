4:set:3

